ci/checks/style.sh

```diff
@@ -13,6 +13,10 @@ LANG=C.UTF-8
 # Activate common conda env
 source activate gdf
 
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -y "clang=11.0.0"
+gpuci_conda_retry install -y "clang-tools=11.0.0"
```

Comment:
These lines should be removed, update the clang version in the integration repo.

Reply:
> update the clang version in the integration repo

Is that something I can do? or does ops need to?

Reply:
You can open a PR against github.com/rapidsai/integration to update this: https://github.com/cwharris/integration/blob/f42792a0bc7e69b414887721f0949f9550b3a6a5/conda/recipes/versions.yaml#L37-L38

Reply:
Removed.